rank/services/search-templates.ts

```diff
@@ -57,7 +61,7 @@ export async function getRemoteTemplates(env: Env): Promise<SearchTemplate[]> {
     remoteTemplates = json.templates.map((template) => {
       const namespace = getNamespaceFromIndexName(template.index)
       return {
-        id: `${env}/${namespace}/${template.index}`,
+        id: `${namespace}/${env}/${template.index}`,
```

Comment:
I feel like the `namespace` value might be redundant here, because we'll never want to run `images` tests against a `works` index, or vice-versa... 

If we know that the index name is `works-2021-07-13`, could we just get the namespace from that? 

Can we safely assume that the index name will always start with `works` or `images`?

Reply:
I didn't want to make that assumption and it makes the code more complex. I'd prefer the split in logic.

We have a set of tests, the use a query from remote or local that is run on an index. `namespace/env/index`.

Creating any other rules has previously landed up with us building in more logic than is necessary. 

The tests will just fail if you get the wrong combo.